libs/features/personalization/personalization.js

```diff
@@ -513,6 +523,10 @@ const getVariantInfo = (line, variantNames, variants, manifestPath, fTargetId) =
   // retro support
   const action = line.action?.toLowerCase()
     .replace('content', '').replace('fragment', '').replace('tosection', '');
+  if (!action) {
+    log('Invalid action found: ', line);
```

Comment:
```suggestion
    log('Row found with empty action field: ', line);
```
The message after this one is for an "invalid" action.  I think this row only triggers when the action is empty.  So let's reword it to differentiate and be more clear.